index.js

```diff
@@ -233,7 +234,7 @@ Toolkit.run(
       tools.log.debug("Something went wrong");
       return tools.exit.failure(err);
     }
-    tools.exit.success("Pushed to remote repository");
+    tools.exit.success("Wrote to README");
```

Comment:
```suggestion
    tools.exit.success("Pushed to remote repository");
```

Isn't it better to keep the logs at different stages as with the case before (being explicit)?